queries/r/highlights.scm

```diff
@@ -0,0 +1,86 @@
+; highlights.scm
+
+(call function: (identifier) @function)
+(namespace_get function: (identifier) @function.method)
```

Comment:
`@method`

Reply:
We get this question over and over. You could create an issue to unifiy the qurey system but right now Emacs, Atom and nvim-treesitter have slightly different queries. The ones that live in their original repo are typically in the Atom-style (even if there is no exact definition what that is). Emacs and Nvim have a bit more highlight groups that often correspond to already existing highlights. That's the reason why we have our own queries here in this repo.

There is also https://github.com/nvim-treesitter/nvim-treesitter/issues/904 and a suggestion to allow more specific quries. E.g. `@function,method` would fallback to `@function` if the more specific one does not exist.

Someone would have to take the effort and change all queries if everyone agrees that having one query for Atom and nvim would be better.

Reply:
See CONTRIBUTING.md

Reply:
> We get this question over and over.

Sorry for trouble. Didn't find anywhere the actual answer to this question, so decided to double check.

Reply:
@echasnovski I was not trying to say that we are annoyed but rather that this is a real issue that many people have (that we should solve?) 

summary of discussion:
- own queries: be more flexible, be more nvim-specific
- use upstream: less maintenance

Maybe we can find a good solution or compromise. PRs and issues to help us resolving that are welcome.